spec/swagger_helper.rb

```diff
@@ -55,6 +55,7 @@
           ApplicationsResponse: APPLICATIONS_RESPONSE[version],
           Application: APPLICATION[version],
           ApplicationAcceptRequest: APPLICATION_ACCEPT_REQUEST,
+          ApplicationAcceptRequestV3: APPLICATION_ACCEPT_REQUEST_V3,
```

Comment:
Suggestion to follow the pattern `ApplicationAcceptRequest: APPLICATION_ACCEPT_REQUEST[version],`